contracts/libraries/GnosisSafeStorage.sol

```diff
@@ -14,8 +14,8 @@ contract GnosisSafeStorage {
     uint256 internal threshold;
 
     // From /GnosisSafe.sol
-    bytes32 internal nonce;
-    bytes32 internal domainSeparator;
+    uint256 internal nonce;
+    bytes32 internal _deprecatedDomainSeparator;
```

Comment:
not the biggest fan of the `_`

Maybe just `deprecatedDomainSeparator` with a comment that links to the changelog section for this: https://github.com/gnosis/safe-contracts/blob/main/CHANGELOG.md#add-migration-example-to-downgrade-from-130-to-120

Reply:
Ok as pointed out in the other PR it seems I named it this way :shrug: ... sorry for complaining :sweat_smile: